### PR TITLE
Export NeDropdownItem type

### PR DESCRIPTION
**Pull request by andre8244**
Export NeDropdownItem type

Other changes:
- Fix hover styles for NeButton
- Remove unused type NePaginatorProps
- Update header comments

### CODE REVIEW

File: src/components/NePaginator.vue
Comment (Tbaile) [inline src/components/NePaginator.vue:1]: Why the change?

**Review by Tbaile — APPROVED** [review]

File: src/components/NePaginator.vue
Comment (andre8244) [reply src/components/NePaginator.vue:1]: If you are referring to the removal of `NePaginatorProps`, this was a type used by the first implementation of table pagination and currently is not used anymore.

Comment (Tbaile) [reply src/components/NePaginator.vue:1]: :ok_hand:
